onnx_chainer/functions/array.py

```diff
@@ -119,8 +119,18 @@ def convert_GetItem(func, opset_version, input_names,
                 'ONNX-Chainer does not accept the type'.format(type(idx)))
 
     gb = onnx_helper.GraphBuilder()
-    output = gb.op('Slice', input_names,
-                   axes=axes, starts=starts, ends=ends)
+    if opset_version == 10:
```

Comment:
This code is not wrong, but to align with other converters, adding `@support(1, 10)` decorator on the converter and the below example looks same with others. (sorry for not intuitive...)

```
if opset_version == 1:
    output = ...
else:
   # for opset10 code
```